package.json

```diff
@@ -67,7 +67,7 @@
     "prettier": "^1.18.2",
     "pretty-quick": "^1.11.1",
     "react-router": "^5.0.1",
-    "react-router-dom": "^5.0.1",
+    "react-router-dom": "^5.2.0",
```

Comment:
Note - this changes nothing. We anyway take the shared `react-router-dom`.

Reply:
(Its still good to have it in there, and even have it in a recent version, but just to re-iterate - we are using a shared lib here)